.gitignore

```diff
@@ -1,3 +1,4 @@
 /target
 /example/toda
-/toda
\ No newline at end of file
+/toda
+.idea
```

Comment:
`.idea` should be added to your global `gitignore`, but not every projects :imp:

Reply:
Okay, I will close it.